code/parachain/runtime/dali/src/migrations.rs

```diff
@@ -19,6 +21,643 @@ impl OnRuntimeUpgrade for SchedulerMigrationV1toV4 {
+							name: Some(
+								BiBoundedAssetName::from_vec(b"PICA KSM LPT".to_vec())
+									.expect("String is within bounds"),
+							),
+							symbol: Some(
+								BiBoundedAssetSymbol::from_vec(b"PICA_KSM_LPT".to_vec())
+									.expect("String is within bounds"),
+							),
+							decimals: Some(12),
+							existential_deposit: 100,
+							ratio: None,
+						},
+					},
+					AssetCreationInput {
+						asset_id: CurrencyId(129),
+						location: Some(XcmAssetLocation(MultiLocation {
+							parents: 1,
+							interior: X2(
+								Parachain(topology::karura::ID),
+								GeneralKey(WeakBoundedVec::force_from(
+									topology::karura::AUSD_KEY.to_vec(),
+									None,
+								)),
+							),
+						})),
+						asset_info: AssetInfo {
+							name: Some(
+								BiBoundedAssetName::from_vec(b"Karura Dollar".to_vec())
+									.expect("String is within bounds"),
+							),
+							symbol: Some(
+								BiBoundedAssetSymbol::from_vec(b"kUSD".to_vec())
+									.expect("String is within bounds"),
+							),
+							decimals: Some(12),
+							existential_deposit: 100_000_000,
+							ratio: Some(rational!(15 / 1_000)),
+						},
+					},
+					AssetCreationInput {
+						asset_id: CurrencyId(130),
+						location: Some(XcmAssetLocation(MultiLocation {
+							parents: 1,
+							interior: X3(
+								Parachain(topology::common_good_assets::ID),
+								PalletInstance(topology::common_good_assets::ASSETS),
+								GeneralIndex(topology::common_good_assets::USDT),
+							),
+						})),
+						asset_info: AssetInfo {
+							name: Some(
+								BiBoundedAssetName::from_vec(b"Tether".to_vec())
+									.expect("String is within bounds"),
+							),
+							symbol: Some(
+								BiBoundedAssetSymbol::from_vec(b"USDT".to_vec())
+									.expect("String is within bounds"),
+							),
+							decimals: Some(6),
+							existential_deposit: 100,
+							ratio: Some(rational!(15 / 1_000_000_000)),
+						},
+					},
+					AssetCreationInput {
+						asset_id: CurrencyId(5),
+						location: Some(XcmAssetLocation(MultiLocation {
+							parents: 0,
+							interior: X1(GeneralIndex(5)),
+						})),
+						asset_info: AssetInfo {
+							name: None,
+							symbol: Some(
+								BiBoundedAssetSymbol::from_vec(b"PBLO".to_vec())
+									.expect("String is within bounds"),
+							),
+							decimals: Some(12),
+							existential_deposit: 100_000_000_000,
+							ratio: Some(rational!(1 / 1)),
+						},
+					},
+					AssetCreationInput {
+						asset_id: CurrencyId(6),
+						location: None,
+						asset_info: AssetInfo {
+							name: None,
+							symbol: Some(
+								BiBoundedAssetSymbol::from_vec(b"ibcDOT".to_vec())
+									.expect("String is within bounds"),
+							),
+							decimals: Some(12),
+							existential_deposit: 214_300_000,
+							ratio: None,
+						},
+					},
+				];
+
+				let total_weight = add_assets_to_storage(assets);
+				StorageVersion::new(1).put::<AssetsRegistry>();
+				total_weight
```

Comment:
```suggestion
				StorageVersion::new(1).put::<AssetsRegistry>();
				add_assets_to_storage(assets)
```

Reply:
nit: no need for the temporary variable, `add_assets_to_storage` doesn't read the storage version so `StorageVersion` can be updated before calling it